test/integration/targets/azure_rm_image/tasks/main.yml

```diff
@@ -51,9 +42,7 @@
       resource_group: "{{ resource_group }}"
       name: "{{ vm_name }}"
       vm_size: Standard_A0
-      storage_account: "{{ storage_name }}"
```

Comment:
why remove the storage account?

Reply:
Since the storage account is not needed here. Creating an image only need a virtual machine, and can be created from disk/blob/snapshot. Here I  remove the storage account and make the image create from the os_disk which can add more test coverage, like send the source as resource id or dict.